Source/ablastr/coarsen/average.cpp

```diff
@@ -105,7 +105,11 @@ namespace ablastr::coarsen::average
                                            "source MultiFab and destination MultiFab have different IndexType");
 
         // Number of guard cells to fill on coarse patch and number of components
-        const amrex::IntVect ngrow = (mf_src.nGrowVect() + 1) / crse_ratio;
+        amrex::IntVect ngrow = mf_src.nGrowVect();
+        for (int l=0; l < AMREX_SPACEDIM; ++l) {
+             if (crse_ratio[l] > 1)  ngrow[l] = (ngrow[l] + 1) / crse_ratio[l];
+        }
+
```

Comment:
```suggestion
        const amrex::IntVect ngrow = (mf_src.nGrowVect() + crse_ratio-1) / crse_ratio; // round up int division
```

Reply:
This also works if crse_ratio is 4 or any integer.

Reply:
Thanks, @WeiqunZhang !